storage/backup/backup-cli/src/storage/command_adapter/command.rs

```diff
@@ -60,7 +60,7 @@ impl SpawnedCommand {
     pub fn spawn(command: Command) -> Result<Self> {
         println!("Spawning {:?}", command);
 
-        let mut cmd = tokio::process::Command::new("sh");
+        let mut cmd = tokio::process::Command::new("bash");
```

Comment:
Looks like on Linux (at least ours)`sh` links to `dash` which doesn't have `-o pipefail`